src/main/java/com/juniorstart/juniorstart/model/JobOffer.java

```diff
@@ -0,0 +1,47 @@
+package com.juniorstart.juniorstart.model;
+
+
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class JobOffer {
```

Comment:
Zamiast robić Getter i Setter to wystarczy @Data 
"@Data is a convenient shortcut annotation that bundles the features of @ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor"
https://projectlombok.org/features/Data

Reply:
+1